timur/lib/client/jsx/components/query/query_slice_control.tsx

```diff
@@ -40,6 +40,7 @@ const QuerySliceControl = ({
           isColumnFilter={true}
           patchClause={handlePatchClause}
           removeClause={() => {}}
+          canRemove={false}
```

Comment:
Seems I don't have my dev environment set up properly to be able to test directly, but the idea makes sense to me and the code seems right, aside from maybe here... should this be *true instead?

Reply:
Note that this funky extra `RemoveIcon` in the wrapping `QuerySliceControl` component is for UI alignment of headings ...

Reply:
This is what the first clause in a filter will look like:

![Screenshot from 2022-09-12 14-31-13](https://user-images.githubusercontent.com/4930129/189729720-d890a653-e8dc-437e-9b7a-f56990de3559.png)


And when you have more than one:

![Screenshot from 2022-09-12 14-31-19](https://user-images.githubusercontent.com/4930129/189729788-92b2aef6-6374-4e9d-bd6d-a271106d85fa.png)

Reply:
Gotcha!